### ISSUE TITLE
Panic running s00002.vtc on travis

### ISSUE THREAD

**Issue by bsdphk** [varnishcache/varnish-cache#1849]
Old ticket imported from Trac
See archived copy here: https://varnish-cache.org/trac/ticket/1849

**Comment by lkarsten** [comment]
Extracted panic from log.txt in the original ticket:
```
Error: Child (53383) Panic at: Fri, 29 Jan 2016 16:22:29 GMT
"Assert error in HSH_DerefBusy(), cache/cache_hash.c line 760:
  Condition((oc)->magic == 0x4d301302) not true.
thread = (cache-worker)
version = varnish-trunk revision 2b8b174
ident = Linux,3.13.0-40-generic,x86_64,-jnone,-smalloc,-smalloc,-hcritbit,epoll
Backtrace:
  0x4328d1: pan_ic+0x151
  0x428ec3: HSH_DerefBusy+0x183
  0x424d4e: VBF_Fetch+0x2ae
  0x4370f8: CNT_Request+0x1a18
  0x44dd73: HTTP1_Session+0xf3
  0x438f1d: SES_Proto_Req+0x5d
  0x4486da: WRK_Thread+0x49a
  0x448b2b: pool_thread+0x2b
  0x2b2baa548e9a: libpthread.so.0(+0x7e9a) [0x2b2baa548e9a]
  0x2b2baa8522ed: libc.so.6(clone+0x6d) [0x2b2baa8522ed]
req = 0x2b2bd8090970 {
  vxid = 1004, step = R_STP_LOOKUP,
  req_body = R_BODY_NONE,
  restarts = 0, esi_level = 0,
  sp = 0x2b2bd4001b70 {
    fd = 13, vxid = 1003,
    client = 127.0.0.1 44939,
    step = S_STP_H1PROC,
  },
  worker = 0x2b2ba97a1c60 {
    stack = {0x2b2ba97a2000 -> 0x2b2ba9796000},
    ws = 0x2b2ba97a1e48 {
      id = \"wrk\",
      {s,f,r,e} = {0x2b2ba97a13f0,0x2b2ba97a13f0,(nil),+2040},
    },
    VCL::method = HIT,
    VCL::return = deliver,
    VCL::methods = {RECV, HASH, HIT},
  },
  ws = 0x2b2bd8090b50 {
    id = \"req\",
    {s,f,r,e} = {0x2b2bd8092950,+72,(nil),+57336},
  },
  http_conn = 0x2b2bd8090a78 {
    fd = 13,
    doclose = NULL,
    ws = 0x2b2bd8090b50,
    {rxbuf_b, rxbuf_e} = {0x2b2bd8092950, 0x2b2bd8092962},
    {pipeline_b, pipeline_e} = {(nil), (nil)},
    content_length = -1,
    body_status = none,
    first_byte_timeout = 0.000000,
    between_bytes_timeout = 0.000000,
  },
  http[req] = 0x2b2bd8090be8 {
    ws[req] = 0x2b2bd8090b50,
    hdrs {
      \"GET\",
      \"/\",
      \"HTTP/1.1\",
      \"X-Forwarded-For: 127.0.0.1\",
    },
  },
  vcl = {
    temp = warm
    srcname = {
      \"<vcl.inline>\",
      \"Builtin\",
    },
  },
  objcore[REQ] = 0x2b2bdc000930 {
    refcnt = 2,
    flags = 0x0,
    exp_flags = 0x1000,
    exp = { 1454084511.931949, 1.000000, 60.000000, 0.000000 }
    objhead = 0x2b2bdc000a90,
    stevedore = 0x2502470 (malloc s0),
  },
  flags = {
  },
},
busyobj = 0x2b2bc8090970 {
  ws = 0x2b2bc80909e8 {
"
```

Gitref for fix in the original trac ticket: a26fdcd6071a6938759039f2513ca624a21b0e72

**Comment by lkarsten** [comment]
Backport review: Looking through my old notes from 4.1.2, this was skipped then after Martin stated that it most likely was a regression from the bo changes in master.

Not scheduling for 4.0 backport review.
